### PR TITLE
[KP-873] New Plugin JSON

### PR DESCRIPTION
**Pull request by graddaniel**
https://kaholo.atlassian.net/browse/KP-873

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,25 +1,52 @@
...
+    filePath,
   } = params;
 
-  let greeting = `Hello ${helloName}!`;
+  const absoluteFilePath = resolve(filePath);
 
-  if (saySecret && !secret) {
-    throw new Error("No secret was provided to say. Please provide a secret or uncheck \"Say Secret\".");
-  }
+  let fileHandle;
```
Comment (xkcm) [inline app.js]: Why not just use `fs.readFile()` ?

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/Kaholo/kaholo-plugin-template#readme",
```
Comment (xkcm) [inline package.json]: Please replace all `kaholo-plugin-template` references to `kaholo-plugin-json`

**Review by xkcm — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (graddaniel) [reply app.js]: Hmm, I've missed that. I though it was a different version's approach.

File: package.json (hunk 2 above)
Comment (graddaniel) [reply package.json]: I forgot to push the changes.

File: app.js (hunk 1 above)
Comment (graddaniel) [reply app.js]: Fixed

**Review by xkcm — APPROVED** [review]

**Pull request merged**